polaris.shopify.com/src/styles/globals.scss

```diff
@@ -135,6 +135,7 @@ html {
 
 html {
   font-size: 106.25%;
+  overflow-y: overlay;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-y suggests `overlay` isn't a valid value. Did you mean `scroll`?